url.bs

```diff
@@ -205,7 +205,7 @@ U+0024 ($) to U+0026 (&amp;), inclusive, U+002B (+), and U+002C (,).
 <p class=note>This is used by <cite>HTML</cite> for
 {{NavigatorContentUtils/registerProtocolHandler()}}, and could also be used by other standards to
 percent-encode data that can then be embedded in a <a for=/>URL</a>'s <a for=url>path</a>,
-<a for=url>query</a>, or <a for=url>fragment</a>. Using it with
+<a for=url>query</a>, or <a for=url>fragment</a>, and is also a <a>valid opaque-host string</a>. Using it with
```

Comment:
```suggestion
<a for=url>query</a>, or <a for=url>fragment</a>, or <a for=/>opaque host</a>. Using it with
```
would be more consistent I think. I did notice while looking at this that valid opaque-host string doesn't say that `%` is fine. Perhaps the forbidden host code points should be two sets after all.

Reply:
Sure, that's good - I like how it now says "embedded ... in an opaque host" and not just "is a valid opaque host".

As for `%`, it says a valid opaque-host string is made of URL Units, which are URL code points (doesn't include `%`) and percent-encoded bytes (which do of course include `%`), so I think it's implied that `%` is fine, but _only_ when used for percent-encoding, and not just generally as part of the host's name.

Reply:
It doesn't read that well to me, but it's a separate issue.

Reply:
Ah: https://github.com/whatwg/url/issues/319.